apps/full-stack-tests/src/hierarchies/learning-snippets/Utils.ts

```diff
@@ -4,17 +4,25 @@
  *--------------------------------------------------------------------------------------------*/
 
 import { collect } from "presentation-test-utilities";
-import { HierarchyNode, HierarchyProvider } from "@itwin/presentation-hierarchies";
+import { HierarchyNode, HierarchyNodeKey, HierarchyProvider } from "@itwin/presentation-hierarchies";
 
 interface NodeDefinition {
+  nodeType: "custom" | Exclude<HierarchyNodeKey, string>["type"];
```

Comment:
why "custom" instead of the actual custom node key?

Reply:
Because here, I wanted to store `node.key.type` and not the value of the key. But custom keys are just strings.